conf/smart-default.xml

```diff
@@ -49,6 +49,16 @@
     </description>
   </property>
 
+  <property>
+    <name>smart.tmp.dir</name>
+    <value></value>
+    <description>
+      This directory is used as a tmp directory for SSM to store tmp files and data.
+      The default path is "/system/ssm", and SSM will ignore files under the tmp directory.
```

Comment:
Please check where /system/ssm is hard-coding. This config should also be used there.